dydx/dydxViews/dydxViews/_v4/TakeProfitStopLoss/Components/dydxGainLossInputViewModel.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  dydxGainLossInputViewModel.swift
+//  dydxUI
+//
+//  Created by Michael Maguire on 4/2/24.
+//  Copyright © 2024 dYdX Trading Inc. All rights reserved.
+//
+
+import SwiftUI
+import PlatformUI
+import dydxFormatter
+import Utilities
+
+public class dydxGainLossInputViewModel: PlatformTextInputViewModel {
+    public init(triggerType: dydxTakeProfitStopLossInputAreaModel.TriggerType, onEdited: ((String?) -> Void)? = nil) {
+        super.init(
+            label: DataLocalizer.shared?.localize(path: triggerType.gainLossInputTitleLocalizerPath, params: nil),
+            placeHolder: dydxFormatter.shared.dollar(number: 0.0, digits: 0),
+            // TODO: replace when percentage input is in abacus
```

Comment:
note that we have made a decision for now to just display the diff between trigger price and gain/loss. Percentage is not yet available in abacus